packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -304,16 +326,20 @@ function HeaderBar(props) {
 						t={props.t}
 					/>
 				)}
+				{props.intercom && (
```

Comment:
Ow, a new prop again 😞 
But sure, as it's done, we can't do it another way

Reply:
Yes, we need to make it modular as the list composition.
But for now, that's not the scope